SConstruct

```diff
@@ -779,6 +788,8 @@ noshell = has_option( "noshell" )
 
 jsEngine = get_option( "js-engine")
 
+rocksdb = get_option( "rocksdb" ) == "on"
```

Comment:
Actually, our current v3.2 implementation just checks the `--rocksdb` option is set and doesn't check values passed with it. I wanted to make the option more strict and behave like `--wiredtiger` or other 'native' Mongo's options. Thoughts?